src/main/java/uk/gov/companieshouse/documentsigningapi/controller/SignDocumentController.java

```diff
@@ -103,7 +103,7 @@ private byte[] addCoverSheetIfRequired(final byte[] document,
                                            final Calendar signingDate) {
         return request.getSignatureOptions() != null &&
                 request.getSignatureOptions().contains(COVER_SHEET_SIGNATURE_OPTION) ?
-                coverSheetService.addCoverSheet(document, request.getCoverSheetData(), signingDate) : document;
+                coverSheetService.addCoverSheet(document, request.getCoverSheetData(), request, signingDate) : document;
```

Comment:
Given we are now passing in `request` we could reduce the number of arguments by removing `request.getCoverSheetData()` as a separate argument.